Game/start_menu.gd

```diff
@@ -85,3 +85,13 @@ func load_level_from_data(level_data):
 	level.load_level(level_data)
 	get_tree().set_deferred("current_scene", level)
 	queue_free()
+	return level
+
+
+func _on_campagin_button_pressed():
```

Comment:
campaign? I see this is misspelled

Reply:
I'm going to make this an issue #97 and not fix it for now